FluentTc.Tests/Locators/GitVCSRootBuilderTests.cs

```diff
@@ -38,41 +38,109 @@ public void SimpleCreation()
             vcsRoot.Project.Id.Should().Be("projectId");
             var properties = vcsRoot.Properties.Property;
             properties.FirstOrDefault(
-                p => p.Name == "agentCleanFilePolicy" && 
-                p.Value == "ALL_IGNORED_UNTRACKED_FILES").Should().NotBeNull();
+                p => p.Name == "agentCleanFilePolicy" &&
+                     p.Value == "ALL_IGNORED_UNTRACKED_FILES").Should().NotBeNull();
             properties.FirstOrDefault(
                 p => p.Name == "agentCleanPolicy" &&
-                p.Value == "ALWAYS").Should().NotBeNull();
+                     p.Value == "ALWAYS").Should().NotBeNull();
             properties.FirstOrDefault(
                 p => p.Name == "authMethod" &&
-                p.Value == "ANONYMOUS").Should().NotBeNull();
+                     p.Value == "ANONYMOUS").Should().NotBeNull();
             properties.FirstOrDefault(
                 p => p.Name == "branch" &&
-                p.Value == "branch").Should().NotBeNull();
+                     p.Value == "branch").Should().NotBeNull();
             properties.FirstOrDefault(
                 p => p.Name == "teamcity:branchSpec" &&
-                p.Value == "branchSpec").Should().NotBeNull();
+                     p.Value == "branchSpec").Should().NotBeNull();
             properties.FirstOrDefault(
                 p => p.Name == "submoduleCheckout" &&
-                p.Value == "CHECKOUT").Should().NotBeNull();
+                     p.Value == "CHECKOUT").Should().NotBeNull();
             properties.FirstOrDefault(
                 p => p.Name == "ignoreKnownHosts" &&
-                p.Value == "true").Should().NotBeNull();
+                     p.Value == "true").Should().NotBeNull();
             properties.FirstOrDefault(
                 p => p.Name == "secure:password" &&
-                p.Value == "password").Should().NotBeNull();
+                     p.Value == "password").Should().NotBeNull();
             properties.FirstOrDefault(
                 p => p.Name == "url" &&
-                p.Value == url.ToString()).Should().NotBeNull();
+                     p.Value == url.ToString()).Should().NotBeNull();
             properties.FirstOrDefault(
                 p => p.Name == "useAlternates" &&
-                p.Value == "true").Should().NotBeNull();
+                     p.Value == "true").Should().NotBeNull();
             properties.FirstOrDefault(
                 p => p.Name == "username" &&
-                p.Value == "username").Should().NotBeNull();
+                     p.Value == "username").Should().NotBeNull();
             properties.FirstOrDefault(
                 p => p.Name == "userNameStyle" &&
-                p.Value == "AUTHOR_NAME").Should().NotBeNull();
+                     p.Value == "AUTHOR_NAME").Should().NotBeNull();
+        }
+
+        [Test]
+        public void SimpleCreationSsh()
```

Comment:
The purpose of this test is to test creation of VCS root with SSH key. So its name should declare it and its Assert part should have a single logical assert that verifies what the test name says.
```C#
public void GetVCSRoot_SshKey_SshKeyCreated()
{
  // Act
  VcsRoot vcsRoot = new GitVCSRootBuilder()
    .AuthMethod(AuthMethod.TeamcitySshKey)
    .UploadedKey("keyName")
    .GetVCSRoot()

  // Assert
  vcsRoot.Properties.Property.ShouldContainSingle(p => p.Name == "authMethod" && 
    p.Value == "TEAMCITY_SSH_KEY");
  vcsRoot.Properties.Property.ShouldContainSingle(p => p.Name == "teamcitySshKey" && 
    p.Value == "keyName");
}

Reply:
Makes sense - testing this case instead of the more general case of creating a VCS Root.  Updated.